File: 1013-fibonacci-number/fibonacci-number.java
class Solution {
    public int fib(int n) {
        int a=0,b=1;
        int next=0;
        if (n==0 || n==1){
            return n;
        }
        for(int i=1;i<n;i++){
            next=a+b;
            a=b;
            b=next;
        }
        return b;
    }
}

File: 1013-fibonacci-number/fibonacci-number.py
class Solution(object):
    def fib(self, n):
        a,b=0,1
        if n==0: return 0
        if n==1: return 1
        next=0
        for i in range(1,n):
            next=a+b
            a=b
            b=next
        return b
        